packages/mdc-fab/_mixins.scss

```diff
@@ -38,6 +38,14 @@
   @include mdc-theme-prop(background-color, $color, $edgeOptOut: true);
 }
 
+@mixin mdc-fab-icon-size($width: 24px, $height: $width) {
```

Comment:
Don't default `width` here; specify it below in `mdc-fab.scss`

Also, this needs to be documented in the README. I'd probably lump it under advanced sass mixins.